cinema/models.py

```diff
@@ -84,23 +92,27 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
-    def clean(self):
-        for ticket_attr_value, ticket_attr_name, cinema_hall_attr_name in [
-            (self.row, "row", "rows"),
-            (self.seat, "seat", "seats_in_row"),
-        ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
+    @staticmethod
+    def validate_row_and_seat(
+            row: int, num_row: int,
+            seat: int, num_seat: int,
+            error_to_raise
```

Comment:
The `error_to_raise` parameter in the `validate_row_and_seat` method is used to raise an exception but is not explicitly typed. Consider specifying its type, such as `Type[Exception]`, for better clarity and type checking.